spec/acceptance/datasets_spec.rb

```diff
@@ -0,0 +1,215 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+require 'helpers/acceptance_spec_helper'
+
+def id_params
+  parameter :id, 'Dataset id in request url', required: true
+end
+
+def body_params
+  parameter :name, 'Name of dataset', scope: :dataset, :required => true
+  parameter :description, 'Description of dataset', scope: :dataset
+end
+
+# https://github.com/zipmark/rspec_api_documentation
+resource 'Datasets' do
+
+  header 'Accept', 'application/json'
+  header 'Content-Type', 'application/json'
+  header 'Authorization', :authentication_token
+
+  let(:format) { 'json' }
+
+  create_entire_hierarchy
+
+  # Create post parameters from factory
+  let(:post_attributes) { FactoryGirl.attributes_for(:dataset, name: 'New Dataset name') }
+
+  ################################
+  # INDEX
+  ################################
+
+  # Expected count is 2 because of the 'default dataset', which is seed data added during a clean install.
+
+  get '/datasets' do
+    let(:authentication_token) { admin_token }
+    standard_request_options(:get, 'INDEX (as admin)', :ok, {expected_json_path: 'data/0/name', data_item_count: 2})
+  end
+
+  # will only return dataset that are created by the user
```

Comment:
So the list of datasets should be available to any user, right? Otherwise, CS user _Joe_ won't be able to list datasets available for them to work on

Reply:
Unlikely that this will be necessary for the CS UI, but for simplicity/consistency fixed so that dataset filter/index are available to everyone. Not available to harvester because it would require specifically whitelisting on the harvester abilities and is not necessary for harvester.